src/main/java/com/google/cloud/pontem/EndToEndHelper.java

```diff
@@ -306,6 +306,7 @@ public static void main(String[] args) throws Exception {
       final boolean shouldFailIfContentAlreadyExists =
           Boolean.valueOf(cmd.getOptionValue("shouldFailIfContentAlreadyExists", "false"));
       final Util util = new Util();
+      final SpannerUtil spannerUtil = new SpannerUtil();
```

Comment:
You are instantiating SpannerUtil in more than one method maybe it'd be best to make it a field if there are no concerns about inconsistent state between with only one instance.